diff --git a/src/c/wrapperinfo.c b/src/c/wrapperinfo.c
index 2461b00..5732357 100644
--- a/src/c/wrapperinfo.c
+++ b/src/c/wrapperinfo.c
@@ -30,6 +30,6 @@ TCHAR *wrapperArch = TEXT("x86");
 TCHAR *wrapperOS = TEXT("linux");
 TCHAR *wrapperReleaseDate = TEXT("20100803");
 TCHAR *wrapperReleaseTime = TEXT("0000");
-TCHAR *wrapperBuildDate = TEXT("20100805");
-TCHAR *wrapperBuildTime = TEXT("0928");
+TCHAR *wrapperBuildDate = TEXT("20111002");
+TCHAR *wrapperBuildTime = TEXT("0000");
 
diff --git a/src/java/org/tanukisoftware/wrapper/WrapperInfo.java b/src/java/org/tanukisoftware/wrapper/WrapperInfo.java
index 17f5ef7..839542a 100644
--- a/src/java/org/tanukisoftware/wrapper/WrapperInfo.java
+++ b/src/java/org/tanukisoftware/wrapper/WrapperInfo.java
@@ -56,8 +56,8 @@ final class WrapperInfo
         Calendar buildTime = Calendar.getInstance();
         try
         {
-            buildDate.setTime( new SimpleDateFormat( "yyyyMMdd" ).parse( "20100805" ) );
-            buildTime.setTime( new SimpleDateFormat( "HHmm" ).parse( "0928" ) );
+            buildDate.setTime( new SimpleDateFormat( "yyyyMMdd" ).parse( "20111002" ) );
+            buildTime.setTime( new SimpleDateFormat( "HHmm" ).parse( "0000" ) );
             
             m_build.set( buildDate.get( Calendar.YEAR ), 
                         buildDate.get( Calendar.MONTH ), 
