Description: correct bashisms in daemon.sh script
Author: tony mancill <tmancill@debian.org>

--- a/src/bin/sh.script.in
+++ b/src/bin/sh.script.in
@@ -294,11 +294,11 @@ if [ "$DIST_OS" = "macosx" ]
 then
     OS_VER=`sw_vers | grep 'ProductVersion:' | grep -o '[0-9]*\.[0-9]*\.[0-9]*\|[0-9]*\.[0-9]*'`
     DIST_ARCH="universal"
-    if [[ "$OS_VER" < "10.5.0" ]]
+    if [ "$OS_VER" \< "10.5.0" ]
     then
         DIST_BITS="32"
     else
-        if [ "X`sysctl -n hw.cpu64bit_capable`" == "X1" ]  
+        if [ "X`sysctl -n hw.cpu64bit_capable`" = "X1" ]
         then
             DIST_BITS="64"
         else
@@ -981,7 +981,7 @@ macosxstart() {
     fi
     # If launchd is set to run the daemon already at Load, we don't need to call start
     getpid
-    if [ "X$pid" == "X" ] ; then
+    if [ "X$pid" = "X" ] ; then
         launchctl start ${APP_PLIST_BASE}
     fi
     
